src/infoset.ts

```diff
@@ -25,11 +25,12 @@ import {
   ensureFile,
   tmpFile,
 } from './utils'
+import * as JSONC from 'jsonc-parser'
 
 // Function to display an infomation message that the infoset file has been created
 // If the user wishes to open the file then they may click the 'Open' button
 async function openInfosetFilePrompt() {
-  let config = JSON.parse(JSON.stringify(getCurrentConfig()))
+  let config = JSONC.parse(JSON.stringify(getCurrentConfig()))
```

Comment:
I don't see the benefit of using JSONC in cases where you're parsing the output of `JSON.stringify` because comments (if there were any) aren't going to survive that operation.